src/Auth0.OidcClient.WPF/WebViewBrowser.cs

```diff
@@ -71,10 +79,21 @@ public Task<BrowserResult> InvokeAsync(BrowserOptions options, CancellationToken
                     tcs.SetResult(new BrowserResult { ResultType = BrowserResultType.UserCancel });
             };
 
+#if NET6_0
+            window.ContentRendered += async (_, e) => 
+            {
+                await webView.EnsureCoreWebView2Async();
+                webView.CoreWebView2.Navigate(options.StartUrl);
+            };
+#endif
```

Comment:
What's the reasoning to stick to `ContentRendered` rather than [the official example for the OidcClient](https://github.com/IdentityModel/IdentityModel.OidcClient.Samples/blob/main/WpfWebView2/WpfWebView2/WpfEmbeddedBrowser.cs)?

Reply:
Because you must call to EnsureCoreWebView2Async before Navigate, and you can't do it immediately after instantiating the window, you must do it in that event as you can see here: [https://stackoverflow.com/a/66834694/583336]( https://stackoverflow.com/a/66834694/583336)

Reply:
Not sure I follow, this is also what happens here: https://github.com/IdentityModel/IdentityModel.OidcClient.Samples/blob/main/WpfWebView2/WpfWebView2/WpfEmbeddedBrowser.cs#L58-L64

Reply:
That was my first approach, but if you do it that way, the call to EnsureCoreWebView2Async() never returns, so I looked for the solution and came across the link above.

Reply:
I will need to look into this as it worked before for me, and i assume it works on the official oidc client sample as well.

Reply:
I'm going to test it.
I'm testing everything with Windows 11.

Reply:
Thanks, same here regarding Windows 11.

Reply:
The problem is that in my computer at least, if you debug in the WebViewBrowser file the call to the EnsureCoreWebView2Async() method, it takes a long time (more than a minute) to return, and that made me think that the call was blocked and that's why I looked for a solution in StackOverflow.

But if you don't debug that file, the call returns immediately, and you don't need to do it inside any event.

Reply:
It's interesting for sure, and something we want to remember for the future if someone encounters the same. Thanks for looking into that.